torch/_inductor/codegen/cpp.py

```diff
@@ -4191,25 +4190,23 @@ def try_loop_split(self, nodes: List[SchedulerNode]):
             assert isinstance(node.node, ir.ComputedBuffer)
             _, original_body, _ = node.node.get_default_sizes_body()
             for name, expr in original_body.indexing_exprs.items():
-                num_div += expr.count(FloorDiv)
-                if num_div > 1:
-                    return nodes
-                if expr.count(FloorDiv) == 1:
-                    div_expr = expr.find(FloorDiv).pop()
-                    split_var = div_expr.args[0]
-                    split_number = div_expr.args[1]
-                    divide_index_name = name
+                for div_expr in expr.find(FloorDiv):
+                    if any(div_expr.has(var) for var in original_body.iter_vars):
+                        num_div += 1
+                    if num_div > 1:
```

Comment:
Is it possible that the div variable and the divider are the same even though there are multiple divs?

Reply:
While I have not encountered this situation, it is possible and I have further relaxed the conditions to support it. Thanks!